projects/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from uuid import uuid4
+from django.contrib.auth.models import User
+
+class Project(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
+    language = models.CharField(max_length=200)
+    concept = models.TextField(blank=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    last_modified = models.DateTimeField(auto_now=True)
```

Comment:
I like the new `Project` model you're building here. I'm a little unclear on what the properties describe; the language is computer language? And concept is like the description? 

At any case, feel free to add more properties that can better describe the resource your API deals with. What's nice about Django is that it will generate additional migrations based on the changes you made to the model, whereas you had to do that manually with knex when we were back in JS land.

Reply:
yeah it's sort of like a todo tracker for things i want to go back and learn/work on. like maybe in javascript i want to practice recursion, go back to redux, look into vue, etc. and maybe in SQL i want to practice queries or assigning foreign keys. so there's the language and then the concept/s within it